extension/build.js

```diff
@@ -0,0 +1,24 @@
+import fs from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
```

Comment:
BTW, you don’t need this hack anymore. Just use `import.meta.dirname`

Reply:
@ai thanks, that's useful! I changed